doc/classes/Array.xml

```diff
@@ -496,37 +509,39 @@
 			<return type="void" />
 			<param index="0" name="position" type="int" />
 			<description>
-				Removes an element from the array by index. If the index does not exist in the array, nothing happens. To remove an element by searching for its value, use [method erase] instead.
-				[b]Note:[/b] This method acts in-place and doesn't return a value.
-				[b]Note:[/b] On large arrays, this method will be slower if the removed element is close to the beginning of the array (index 0). This is because all elements placed after the removed element have to be reindexed.
+				Removes the element from the array at the given index ([param position]). If the index is out of bounds, this method fails.
+				If you need to return the removed element, see [method pop_at]. To remove an element by value, use [method erase] instead.
+				[b]Note:[/b] This method shifts every element's index after [param position] back, which may have a noticeable performance cost, especially on larger arrays.
 			</description>
 		</method>
 		<method name="resize">
 			<return type="int" />
 			<param index="0" name="size" type="int" />
 			<description>
-				Resizes the array to contain a different number of elements. If the array size is smaller, elements are cleared, if bigger, new elements are [code]null[/code].
+				?? Resizes the array to contain a different number of elements. If [param size] is smaller than the array's current size, the elements at the end are removed. Otherwise, the new elements are [code]null[/code].
+				?? Sets the array's number of elements to [param size]. If [param size] is smaller than the array's current size, the elements at the end are removed. If [param size] is greater, new [code]null[/code] elements are added.
```

Comment:
The latter revision is more correct and less repetitive, but the former revision does not sound too bad, either.